jaseci_core/jaseci/actions/remote_actions.py

```diff
@@ -71,7 +71,7 @@ def gen_api_service(app, func, act_group, aliases, caller_globals):
 
     # Create duplicate funtion for api endpoint and inject in call site globals
     @app.post(f"/{func.__name__}/")
-    def new_func(params: model):
+    def new_func(params: model = model.construct()):
```

Comment:
@marsninja can you take a quick look at this? The previous test failure was because when we used `model()` it was failing pedantic validation because `model` has required fields which we are not setting (and can't set here). So I am using `construct` to create a model instance without validation here. 

I tested with several remote actions, with and without required parameters and its working.

Reply:
Well if it works it works :-)